gen-web/src/index.js

```diff
@@ -454,32 +488,38 @@
     } else if (stepTracker === 3) {
       /* Check for download*/
       verifyDownloadComplete()
-    } else if (stepTracker === 4) {
-      inlineVariables.downloadFileName.innerHTML = genConfigFileName(deviceNumber, deviceID)
-    } else if (stepTracker === 4) {
-      /* Display back User Email on Page 4 for visual email verification */
-      inlineVariables.emailPlaceholder.innerHTML = user.email || console.error('User Email not found. Config may be corrupted.')
+    } else if (stepTracker === 5) {
+      // TODO put this back inlineVariables.downloadFileName.innerHTML = genConfigFileName(deviceNumber, deviceID)
+      inlineVariables.downloadFileName.innerHTML = "whatever.jsoon"
     }
```

Comment:
I removed this branch entirely as in the existing code it would never be called (if we're on step 4 we will run the branch starting on line 457 above).